content/en/getting_started/containers/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Getting Started with Containers
+kind: documentation
+---
+
+{{< whatsnext desc=" " >}}
```

Comment:
This page looks a bit strange in the preview. Is there a way to get some space between the link and the tag?